src/current/v23.1/restore.md

```diff
@@ -22,7 +22,7 @@ For details on restoring across versions of CockroachDB, see [Restoring Backups
 ## Considerations
 
 - `RESTORE` cannot restore backups made by newer versions of CockroachDB.
-- `RESTORE` only supports backups taken on a cluster on a specific major version into a cluster that is on the same version or the next major version. 
+- `RESTORE` only supports backups taken on a cluster on a specific major version into a cluster that is on the same version or the next major version.
```

Comment:
can link out to https://www.cockroachlabs.com/docs/stable/restoring-backups-across-versions.html

Reply:
Sure, I can add this link as an opportunistic edit not related to the PR.